andela/team-odd-bn-backend
# #169258631 user should be able to login

### What does this PR do?
Users should be able to login into the system.

#### Description of Task to be completed?
Users cannot access the application without logging in without providing an email and password combination.
**A user**  can click on the "Login" button.
**So that**  they can login to the application

#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-backend.git`
- `cd team-odd-bn-backend`
- `npm install`
- `npm run dev-start` check API in development 
- `npm test` testing API

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#169258631](https://www.pivotaltracker.com/story/show/169258631)

#### Screenshots (if appropriate)
Development 
<img width="1440" alt="Screen Shot 2019-11-13 at 10 49 20" src="https://user-images.githubusercontent.com/56690376/68747508-593c1600-0603-11ea-96d7-0ec6b09847c1.png">

Documentation
<img width="1440" alt="Screen Shot 2019-11-12 at 07 48 30" src="https://user-images.githubusercontent.com/56690376/68645502-e9edf580-0520-11ea-88b3-7bc793a279a9.png">


#### Questions:
N/A
